RUFAS/output_manager.py

```diff
@@ -236,7 +254,7 @@ def setup_pool_overflow_control(
             f"{self.available_memory} Bytes.\n"
         )
 
-        if save_chunk_threshold_call_count > 0:
+        if save_chunk_threshold_call_count and save_chunk_threshold_call_count > 0:
```

Comment:
Addresses mypy error.

Reply:
```suggestion
        if save_chunk_threshold_call_count is not None and save_chunk_threshold_call_count > 0:
```
I think it would be better to make this condition a little more explicit, I had to stop and think through it for a sec before I understood what it was doing.